### PR TITLE
Fix Preference Page Icons

### PR DESCRIPTION
**Pull request by niyibi250**
### What does this PR do?
This PR updates the trainee preference settings page to maintain consistency by replacing all buttons or clickables with Chevron icons.
### Description of Task to be completed?

- Replace the forward arrow icons on the settings page with Chevron icons for better UI consistency.
- Ensure that the new Chevron icons are interactive and reflect the current colorScheme (light or dark mode).
- Update the push notification section to include expand/collapse functionality, toggling between a downward and forward Chevron icon.

### How should this be manually tested?

1. Navigate to the trainee preference settings page.

2. Verify that all forward arrows have been replaced by Chevron icons.

3. Test the push notification section:

- Click the Chevron icon to expand the section and reveal the switch.
- Click the Chevron icon again to collapse the section.

4. Ensure the Chevron icons adjust their color dynamically based on the active colorScheme.

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker/Trello stories?
N/A
### Screenshots

image

image

### CODE REVIEW

**Review by MugemaneBertin2001 — CHANGES_REQUESTED** [review]
Screenshot 2024-11-20 100951
1. Use standard react native icons framework instead of images in assets
2. Check settings we have on web as that are written
3. Change to toggle on privacy and login history just as that are on web

**Review by MugemaneBertin2001 — CHANGES_REQUESTED** [review]
Screenshot 2024-11-20 132505
@niyibi250  check on these toggler on web

File: assets/Preference_icons/down_arrow.png
Comment (pacifiquemboni) [inline assets/Preference_icons/down_arrow.png:1]: Good work @niyibi250 
but i think iti is not advisable to use static files especially icons

**Review by pacifiquemboni — COMMENTED** [review]
Good work @niyibi250 
but i think iti is not advisable to use static files especially icons

**Review by pacifiquemboni — COMMENTED** [review]
Resolve conflicting files

File: app/auth/login/index.tsx [hunk 1]
```diff
@@ -84,8 +85,10 @@ export default function SignInOrganization() {
 
   const onSubmit = async (userInput: any) => {
     try {
+      console.log(userInput)
```
Comment (MugemaneBertin2001) [inline app/auth/login/index.tsx]: remove these console logs

File: app/auth/two-factor/index.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        },
+      });
+      setLoading(false)
+    } catch {
+      setLoading(false);
+    }
+  };
+
+  const handleInputChange = (index: number, value: string) => {
+    console.log('typeeeee')
```
Comment (MugemaneBertin2001) [inline app/auth/two-factor/index.tsx]: This is also needs to be removed

File: assets/Preference_icons/down_arrow.png
Comment (MugemaneBertin2001) [reply assets/Preference_icons/down_arrow.png:1]: Try to use this one https://reactnativeelements.com/docs/customization/themeprovider

**Review by MugemaneBertin2001 — CHANGES_REQUESTED** [review]

File: %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver
Comment (Christian-Ishimwe) [inline %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver:1]: delete this file

**Review by Christian-Ishimwe — CHANGES_REQUESTED** [review]

File: %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver
Comment (fredshema) [inline %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver:1]: This file is not needed

**Review by fredshema — CHANGES_REQUESTED** [review]

**Comment by fredshema** [comment]
Also fix conflicts

File: app/auth/two-factor/index.tsx [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+`;
+
+export const RESEND_OTP = gql`
+  mutation ResendOTP($email: String!) {
+    resendOTP(email: $email) {
+      message
+    }
+  }
+`;
+
```
Comment (fredshema) [inline app/auth/two-factor/index.tsx]: Can these be moved to their own mutation files

**Review by fredshema — CHANGES_REQUESTED** [review]
Make sure to also run format before pushing

**Review by MugemaneBertin2001 — APPROVED** [review]
This looks Good to me

**Review by fredshema — APPROVED** [review]

**Pull request merged**